tests/src/dir.targets

```diff
@@ -64,14 +64,22 @@
   <!-- If we are a run-only, that depends on another project, this is the "Build" we use. I.e. build all dependency projects, absolutely.
   -->
 
-  <Target Name="Build" Condition="('$(CLRTestKind)'=='RunOnly') And ('$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)')">
+  <PropertyGroup> 
+    <BuildAllProjects Condition=" '$(BuildAllProjects)' == ''">false</BuildAllProjects>
+    <_CLRTestBuildPrioritySatisfied Condition=" '$(_CLRTestBuildPrioritySatisfied)' == ''">false</_CLRTestBuildPrioritySatisfied>
+    <_CLRTestBuildPrioritySatisfied Condition="'$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)'">true</_CLRTestBuildPrioritySatisfied>
+    <_WillCLRTestProjectBuild Condition="Exists('$(ToolsDir)Build.Common.targets') And $(_CLRTestCompilesSource)">true</_WillCLRTestProjectBuild>
```

Comment:
Isn't it safe to assume that this file should always be present. why is there a special check for its presence? Is there a scenario where this file is absent and that is fine for the build to succeed?